hledger-lib/Hledger/Data/AccountBalance.hs

```diff
@@ -0,0 +1,159 @@
+{-# LANGUAGE CPP #-}
+{-|
+
+
+An 'AccountBalance' consists of a historical balance, along with balances
```

Comment:
Maybe clarify that this module contains code for both the AccountBalances and AccountBalance types, which have separate purposes.

Reply:
I'll break them apart into separate modules.